flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/AbstractSlotSharingStrategy.java

```diff
@@ -95,15 +96,15 @@ public void notifySchedulingTopologyUpdated(
      * topologically sorted.
      */
     @Nonnull
-    static LinkedHashMap<JobVertexID, List<SchedulingExecutionVertex>> getExecutionVertices(
-            SchedulingTopology topology) {
-        final LinkedHashMap<JobVertexID, List<SchedulingExecutionVertex>> vertices =
-                new LinkedHashMap<>();
+    static <T> LinkedHashMap<JobVertexID, List<T>> getExecutionVertices(
```

Comment:
Introducing a function parameter makes it difficult to read. Can add params comments for easier reading?